pkg/webhook/resources/setting/validator.go

```diff
@@ -64,6 +65,7 @@ const (
 	labelAppNameValueGrafana          = "grafana"
 	labelAppNameValueImportController = "harvester-vm-import-controller"
 	maxTTLDurationMinutes             = 52560000 //specifies max duration allowed for kubeconfig TTL setting, and corresponds to 100 years
+	MinallocatableIPAddrs             = 16
```

Comment:
```suggestion
	minAllocatableIPAddrs             = 16
```
Nit: use camel case for consistency and it doesn't look like this variable needs to be exported.

Reply:
Thanks for the explanation on where the `16` comes from. Let me know when the new changes are ready.